gf-webapp-admin/src/component/registration/Registration.module.scss

```diff
@@ -28,7 +28,7 @@
 }
 
 .registrationBtn {
-  padding: 5px 10px;
+  padding: 5px 20px;
```

Comment:
you already have button in library. Why do u create another style here?

Reply:
Done